scalableminds/webknossos
# Task creation info

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Shows a list of taskIDs after task creation

Steps to test:
- Create tasks from nmls
[2012-06-28_Cortex__explorational__sboy__666953.nml.zip](https://github.com/scalableminds/webknossos/files/865403/2012-06-28_Cortex__explorational__sboy__666953.nml.zip)
- Check the modal

Issues:
- fixes https://discuss.webknossos.org/t/show-task-id-affiliation-for-created-tasks/56/10

------
